### PR TITLE
Convert lisk-constant mocha tests to jest - Closes #4675

### LINKED ISSUE #4675: Convert lisk-constants mocha test to jest
### Description

Convert the tests in `lisk-constants` library from `mocha` to `jest`.

### Motivation

Consistent use of `jest` testing.

### Acceptance Criteria

`jest`

### PR DESCRIPTION
**Pull request by mitsuaki-u**
### What was the problem?

Tests were written in `mocha`.

### How did I solve it?

Converted to `jest`.

### How to manually test it?

`npm run test`

### Review checklist

- [ ] The PR resolves #4675 
- [ ] All new code is covered with unit tests
- [ ] Relevant integration / functional tests are added
- [ ] Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
- [ ] Documentation has been added/updated

### CODE REVIEW

File: templates/tsconfig.json.tmpl [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		"declaration": true,
 		"outDir": "dist-node",
 		"rootDir": "./src",
-		"types": ["node", "mocha", "sinon", "chai"]
+		"types": ["node"]
```
Comment (shuse2) [inline templates/tsconfig.json.tmpl]: this should be kept until all the elements are changed

File: templates/jest.config.js.tmpl [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
 	testMatch: ['<rootDir>/test/**/?(*.)+(spec|test).+(ts|tsx|js)'],
-	setupFilesAfterEnv: ['<rootDir>/test/_setup.js'],
+	setupFilesAfterEnv: ['<rootDir>/test/setup.js'],
```
Comment (shuse2) [inline templates/jest.config.js.tmpl]: this shouldn't be changed

File: elements/lisk-constants/test/index.spec.ts [hunk 3]
```diff
@@ -27,38 +28,38 @@ import {
...
 	});
 
 	it('MAX_TRANSACTION_AMOUNT should be a string', () => {
-		return expect(MAX_TRANSACTION_AMOUNT).to.be.a('string');
+		return expect(MAX_TRANSACTION_AMOUNT).toBeString();
 	});
 
 	it('TESTNET_NETHASH should be a string', () => {
-		return expect(TESTNET_NETHASH).to.be.a.hexString;
+		return expect(TESTNET_NETHASH).toBeString();
```
[the remark this answers is not in the document]
Comment (mitsuaki-u) [reply elements/lisk-constants/test/index.spec.ts]: Hmm it doesn't pass using `.toBeHexadecimal` when it should

[the remark this answers is not in the document]
Comment (valamidev) [reply elements/lisk-constants/test/index.spec.ts]:
```
const reg = /[0-9A-Za-z]/g;

expect(reg.test(TESTNET_NETHASH)).toBe(true);
```

**Pull request merged**